setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+from setuptools import setup
+
+
+with open(os.path.join(os.path.dirname(__file__), "README.rst")) as readme:
```

Comment:
getting the directory that `setup.py` is in is repeated below. Maybe do once and use in both places?